### PR TITLE
Update Next.js tests to work with Node 18

### PR DESCRIPTION
**Pull request by khanayan123**
### What does this PR do?
passes --openssl-legacy-provider as NODE_OPTIONS for next.js tests

### Motivation
Next.js tests for versions <13.2 failing on Node.js 18

### CODE REVIEW

**Review by rochdev — DISMISSED** [review]

File: .github/workflows/appsec.yml [hunk 1]
```diff
@@ -202,27 +202,15 @@ jobs:
...
-        include:
-          - node-version: 18
-            range: '>=13.2'
     runs-on: ubuntu-latest
     env:
       PLUGINS: next
-      RANGE: ${{ matrix.range }}
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/actions/testagent/start
```
Comment (Qard) [inline .github/workflows/appsec.yml:218]: Was deleting the testagent parts intentional?

**Review by Qard — APPROVED** [review]
LGTM.

But is there no way to get this working without the flag? It's generally strongly recommended not to use it for production apps as it disables a bunch of security features which therefore changes behaviour. _Probably_ not a big deal for an instrumentation test app, but if we can make it work without needing the flag that _would_ be better as it more closely mirrors how an actual app would behave. I'm guessing it's just an incompatibility with the older versions of Next.js and not really anything we can do about it?

**Comment by khanayan123** [comment]
> LGTM.
> 
> But is there no way to get this working without the flag? It's generally strongly recommended not to use it for production apps as it disables a bunch of security features which therefore changes behaviour. _Probably_ not a big deal for an instrumentation test app, but if we can make it work without needing the flag that _would_ be better as it more closely mirrors how an actual app would behave. I'm guessing it's just an incompatibility with the older versions of Next.js and not really anything we can do about it?

I believe the only way to make it work is through the flag (https://github.com/webpack/webpack/issues/15900) This is more for our CI build, so that we don't have to hardcode node 16 inorder to test certain versions of next.js

File: .github/workflows/appsec.yml (hunk 1 above)
Comment (khanayan123) [reply .github/workflows/appsec.yml:218]: No it wasn't repushed

**Review by rochdev — APPROVED** [review]

**Review by uurien — APPROVED** [review]

**Pull request merged**